darwin/DarwinProcess.h

```diff
@@ -29,6 +35,10 @@ Process* DarwinProcess_new(const Settings* settings);
 
 void Process_delete(Object* cast);
 
+void DarwinProcess_writeField(const Process* this, RichString* str, ProcessField field);
+
+long DarwinProcess_compare(const void* v1, const void* v2);
+
```

Comment:
No need to have those two declarations externally visible.

You can just move the initialization of the class info structure to the bottom of `darwin/DarwinProcess.c` (compare with other platforms like `linux` in `linux/LinuxProces.c`).

Reply:
Done.